src/lib/components/Button.svelte

```diff
@@ -9,7 +9,7 @@
 </script>
 
 <button
-    class="rounded-full bg-black px-4 py-3 text-center font-inter text-csi-white dark:bg-csi-blue dark:text-csi-black"
+    class="rounded-full bg-black px-4 py-3 text-center font-inter text-csi-white transition-colors duration-300 ease-in-out hover:bg-csi-blue dark:bg-csi-blue dark:text-csi-black dark:hover:bg-csi-white"
```

Comment:
I mentioned this in #101 (which admittedly came very late), but I'm of the opinion that hover transitions are best when they are barely noticeable. 

With that said, I believe a transition duration of 300ms for a button tends to be bordering on too much. I personally think the default duration of [TailwindCSS transitions (150ms)](https://tailwindcss.com/docs/transition-property) are fairly sane and often better to keep unless you are aiming to do something specific. This comment is also directed to every component with `duration-300`.

Reply:
Shortened all the durations! Though I noticed that the NavBar durations that were added a bit ago are also around 200-300ms; kept those the same for now but I can shorten those too if you think that looks better as well.